releases/v16.rst

```diff
@@ -0,0 +1,28 @@
+#######################
+LSST Stack 16.0 Release
+#######################
+
+Version 16.0 of the LSST Science Pipelines is now available. It is based on 2018 weekly 10.
+
+LSST Pipelines
+==============
+
+Full details of the LSST Science Pipelines 15.0 release can be found
```

Comment:
"LSST Science Pipelines 16.0"

Reply:
I missed this update. Al the rest should be fine:
- v16_0 in the toctree
- 16.0 defined as current (releases/notes.rst, I did not find other places to change it)